linkis-engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/executor/HiveEngineConnExecutor.scala

```diff
@@ -414,6 +416,7 @@ class HiveEngineConnExecutor(
     singleSqlProgressMap.clear()
     Utils.tryAndWarnMsg(sessionState.close())("close session failed")
     super.close()
+    killTask("close")
```

Comment:
"close"?

Reply:
Maybe can remove it, for the super method has invoked the kill task.